kevm-pyk/src/kevm_pyk/kproj/evm-semantics/gas.md

```diff
@@ -102,23 +102,23 @@ module GAS-FEES
     imports GAS-SYNTAX
     imports SCHEDULE
 
-    syntax Gas ::= Cgascap        ( Schedule , Gas , Gas , Int )         [klabel(Cgascap),         function, total, smtlib(gas_Cgascap_Gas)  ]
-    syntax Int ::= Cgascap        ( Schedule , Int , Int , Int )         [klabel(Cgascap),         function, total, smtlib(gas_Cgascap_Int)  ]
-                 | Csstore        ( Schedule , Int , Int , Int )         [klabel(Csstore),         function, total, smtlib(gas_Csstore)      ]
-                 | Rsstore        ( Schedule , Int , Int , Int )         [klabel(Rsstore),         function, total, smtlib(gas_Rsstore)      ]
-                 | Cextra         ( Schedule , Bool , Int , Bool )       [klabel(Cextra),          function, total, smtlib(gas_Cextra)       ]
-                 | Cnew           ( Schedule , Bool , Int )              [klabel(Cnew),            function, total, smtlib(gas_Cnew)         ]
-                 | Cxfer          ( Schedule , Int )                     [klabel(Cxfer),           function, total, smtlib(gas_Cxfer)        ]
-                 | Cmem           ( Schedule , Int )                     [klabel(Cmem),            function, total, smtlib(gas_Cmem), memo   ]
-                 | Caddraccess    ( Schedule , Bool )                    [klabel(Caddraccess),     function, total, smtlib(gas_Caddraccess)  ]
+    syntax Gas ::= Cgascap        ( Schedule , Gas , Gas , Int ) [symbol(Cgascap_Gas), overload(Cgascap), function, total, smtlib(gas_Cgascap_Gas)]
+    syntax Int ::= Cgascap        ( Schedule , Int , Int , Int ) [symbol(Cgascap_Int), overload(Cgascap), function, total, smtlib(gas_Cgascap_Int)]
+    syntax Int ::= Csstore        ( Schedule , Int , Int , Int )         [klabel(Csstore),        function, total, smtlib(gas_Csstore)       ]
```

Comment:
Why has this new `syntax Int ::= ...` been added?

Reply:
Just to indicate to separate the overloads and the non-aligned attributes a bit. I can remove it though.